app/views/play/common/LadderSubmissionView.coffee

```diff
@@ -77,7 +77,7 @@ module.exports = class LadderSubmissionView extends CocoView
     if @level.isType('ladder') and tempSession.get('team') is 'ogres'
       code = tempSession.get('code') ? {'hero-placeholder': {plan:''}, 'hero-placeholder-1': {plan: ''}}
       tempSession.set('team', 'humans')
-      code['hero-placeholder'] = code['hero-placeholder-1']
+      code['hero-placeholder'] = JSON.parse(JSON.stringify(code['hero-placeholder-1']))
```

Comment:
Is this just to make a recursive copy? Seems inefficient; `_.cloneDeep`  is better for that. But here I think this is a single level object, so a recursive copy isn't needed and you could use `_.clone` or `Object.assign {}, code['hero-placeholder-1']`.